api/docs/release.dox

```diff
@@ -148,6 +148,7 @@ compatibility changes:
 
 Further non-compatibility-affecting changes include:
 
+ - Added the function instr_is_xsave().
```

Comment:
nit: Normally we append new entries to the bottom (unless they're large features we want to call out)

Reply:
Sure, done.